Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json

```diff
@@ -28,5 +28,28 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
+    "x2_fields": "action_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "childprocess",
+          "type": "multiSelect"
+        },
+        {
+            "cliName": "digitalguardianprocessname",
+            "type": "shortText"
+        },
+        {
+            "cliName": "processname",
```

Comment:
Process name should be actor_process_image_name (Initiated By) not action_process_image_name (Target process name)